test/connectivity/scripts/connectivityTest.sh

```diff
@@ -175,6 +228,8 @@ function process_args() {
                 '-verificationDelay' ) saveNextArg=18;;
                 '-upstreamProtocol' ) saveNextArg=19;;
                 '-deploymentTestUpdatePeriod' ) saveNextArg=20;;
+                '-timeForReportingGeneration' ) saveNextArg=21;;
+                '-waitForTestComplete' ) WAIT_FOR_TEST_COMPLETE=1;;
```

Comment:
nit: setting a precedent for bypassing the arg parsing pattern might decrease readability in the future

Reply:
user should just pass -waitForTestComplete argument only when run the script; therefore we need to assign the value here.